netebpfext/net_ebpf_ext_xdp.c

```diff
@@ -51,9 +51,8 @@ const net_ebpf_extension_wfp_filter_parameters_t _net_ebpf_extension_xdp_wfp_fil
 
 typedef struct _net_ebpf_extension_xdp_wfp_filter_context
 {
-    const net_ebpf_extension_hook_client_t* client_context;
+    net_ebpf_extension_wfp_filter_context_t;
```

Comment:
not a big deal, but I prefer always using named fields so you don't need to have type casts, but can use CONTAINING_RECORD() to go from net_ebpf_extension_wfp_filter_context_t to net_ebpf_extension_xdp_wfp_filter_context_t, and just use the field by name to go in the other direction.  This is not blocking, I will approve either way but wanted to pass on what I think is best practice in coding conventions, since it's much more typesafe than using casts which can be bug prone.